.github/workflows/emscripten.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           fetch-tags: true
           persist-credentials: false
 
-      - uses: pypa/cibuildwheel@ee63bf16da6cddfb925f542f2c7b59ad50e93969  # v2.22.0
+      - uses: pypa/cibuildwheel@v2.23.0
```

Comment:
Please use a hash here. There is another cibuildwheel invocation you can copy from.

Reply:
Ah, sorry – I thought I pushed that but I didn't. Applied with a rebase. Thanks!